packages/publisher/package.json

```diff
@@ -10,13 +10,16 @@
     "@definitelytyped/header-parser": "workspace:*",
     "@definitelytyped/retag": "workspace:*",
     "@definitelytyped/utils": "workspace:*",
+    "libnpmpublish": "^9.0.3",
```

Comment:
I thought this change would let us stop depending on npm-registry-publish, but it just adds the new dependency.

Reply:
Yeah, it's half a change; this PR removed some of the registry dep but not all.